### PR TITLE
Multimedia Player: WGAG 2.1 & 2.2 AA Assessment

### PR DESCRIPTION
**Pull request by BrahimMahadi**
### This pull request includes the WCAG 2.1 & 2.2 Accessibility assessments done for the multimedia plugins (audio only & video) 

General checklist
- [ ] Updated multimedia documentation to include the accessibility assessment reports
- [ ] Plugin was tested against WCAG for accessibility
- [ ] Documentation is bilingual

### CODE REVIEW

File: site/pages/docs/ref/multimedia/multimedia-en.hbs [hunk 1]
```diff
@@ -28,6 +28,16 @@
 	</ul>
 </section>
 
+<section>
+<h2>Evaluation and report</h2>
+<ul>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-en.html">English Audio Only Report</a></li>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-fr.html">French Audio Only Report</a></li>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-en.html">English Video Report</a></li>
+	<li>Accessibility (SC WCAG 2.1/2.2 Level A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-fr.html">French Video Report</a></li>
```
Comment (duboisp) [inline site/pages/docs/ref/multimedia/multimedia-en.hbs:37]: Le caractère `&` dois être échapé. `&amp;`

File: site/pages/docs/ref/multimedia/multimedia-fr.hbs [hunk 2]
```diff
@@ -30,6 +30,16 @@
 	</ul>
 </section>
 
+<section>
+<h2>Évaluation et rapport</h2>
+<ul>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-en.html">Anglais Audio Seulement Rapport</a></li>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-audio-fr.html">Français Audio Seulement Rapport</a></li>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-en.html">Anglais Video Rapport</a></li>
+	<li>Accessibilité (CS WCAG 2.1/2.2 Niveau A & AA)- <a href="../../../demos/multimedia/reports/a11y-1-video-fr.html">Français Video Rapport</a></li>
```
Comment (duboisp) [inline site/pages/docs/ref/multimedia/multimedia-fr.hbs:39]: Le caractère `&` dois être échapé. `&amp;`

**Review by duboisp — CHANGES_REQUESTED** [review]
The audio and video assessment are missing a few SC to be evaluated. 

@BrahimMahadi let's connect on this one. we will go through it together.

**Review by duboisp — CHANGES_REQUESTED** [review]
@BrahimMahadi please update your assessment according to the new methodologies and the revised vocabulary terms.

File: site/pages/docs/ref/multimedia/multimedia-fr.hbs (hunk 2 above)
Comment (BrahimMahadi) [reply site/pages/docs/ref/multimedia/multimedia-fr.hbs]: Addressed in commit "Updated assessment format"

File: site/pages/docs/ref/multimedia/multimedia-en.hbs (hunk 1 above)
Comment (BrahimMahadi) [reply site/pages/docs/ref/multimedia/multimedia-en.hbs]: Addressed in commit "Updated assessment format"

File: src/plugins/multimedia/reports/a11y-1-audio-en.hbs [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+{
+	"title": "Accessibility assessment #2 - Multimedia Player - Audio-only",
+	"language": "en",
+	"tag": "multimedia",
+	"parentdir": "multimedia",
+	"parentPage": "Multimedia Player - Audio-only",
+	"parentPageURL": "multimedia",
+	"altLangPage": "a11y-1-audio-fr.html",
+	"dateModified": "2023-06-21",
+	"layout": "acr-en.hbs",
```
Comment (duboisp) [inline src/plugins/multimedia/reports/a11y-1-audio-en.hbs:11]:
```suggestion
	"layout": "assessment_wrote_en-en.hbs",
```

File: src/plugins/multimedia/reports/a11y-1-audio-fr.hbs [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+{
+	"title": "Assessment d'accessibilité #2 - Lecteur multimédia - Audio seulement",
+	"language": "fr",
+	"tag": "multimedia",
+	"parentdir": "multimedia",
+	"parentPage": "Lecteur multimédia - Audio seulement",
+	"parentPageURL": "multimedia",
+	"altLangPage": "a11y-1-audio-en.html",
+	"dateModified": "2023-06-21",
+	"layout": "acr-en.hbs",
```
Comment (duboisp) [inline src/plugins/multimedia/reports/a11y-1-audio-fr.hbs:11]:
```suggestion
	"layout": "assessment_wrote_en-fr.hbs",
```

File: src/plugins/multimedia/reports/a11y-1-video-en.hbs [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+{
+	"title": "Accessibility assessment #2 - Multimedia Player - Video",
+	"language": "en",
+	"tag": "multimedia",
+	"parentdir": "multimedia",
+	"parentPage": "Multimedia Player - Video",
+	"parentPageURL": "multimedia",
+	"altLangPage": "a11y-1-video-fr.html",
+	"dateModified": "2023-06-20",
+	"layout": "acr-en.hbs",
```
Comment (duboisp) [inline src/plugins/multimedia/reports/a11y-1-video-en.hbs:11]:
```suggestion
	"layout": "assessment_wrote_en-en.hbs",
```

File: src/plugins/multimedia/reports/a11y-1-video-fr.hbs [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+{
+	"title": "Assessment d'accessibilité #2 - Lecteur multimédia - Vidéo",
+	"language": "fr",
+	"tag": "multimedia",
+	"parentdir": "multimedia",
+	"parentPage": "Lecteur multimédia - Vidéo",
+	"parentPageURL": "multimedia",
+	"altLangPage": "a11y-1-video-en.html",
+	"dateModified": "2023-06-20",
+	"layout": "acr-fr.hbs",
```
Comment (duboisp) [inline src/plugins/multimedia/reports/a11y-1-video-fr.hbs:11]:
```suggestion
	"layout": "assessment_wrote_en-fr.hbs",
```

**Review by duboisp — CHANGES_REQUESTED** [review]
It's look great.

We will be able to merge after the layout name are updated for the assessment report.

File: src/plugins/multimedia/reports/a11y-1-video-fr.hbs (hunk 6 above)
Comment (BrahimMahadi) [reply src/plugins/multimedia/reports/a11y-1-video-fr.hbs]: Fixed in commit "Updated layout names"

File: src/plugins/multimedia/reports/a11y-1-audio.json
Comment (duboisp) [inline src/plugins/multimedia/reports/a11y-1-audio.json:1]: The N/A items need a quick review

**Review by duboisp — CHANGES_REQUESTED** [review]

File: src/plugins/multimedia/reports/a11y-1-audio.json
Comment (BrahimMahadi) [reply src/plugins/multimedia/reports/a11y-1-audio.json:1]: Changes made in the commit "Quick review performed".

**Review by duboisp — APPROVED** [review]
Review and tested locally, work as expected

**Pull request merged**